common/services/app/civic-uk/scripts.tsx

```diff
@@ -10,24 +45,47 @@ const CivicUK = () => (
             product: 'COMMUNITY',
             apiKey: '5a1b492f47fac442240976a4f991ad92278ef15a',
             product: 'pro',
-            branding: { removeIcon: true }, 
+            initialState: 'notify',
+            layout: 'popup',
+            setInnerHTML: true,
+            closeStyle: 'button',
+            settingsStyle: 'button',
+            notifyDismissButton: false,
+            acceptButton: false,
+            rejectButton: false,
```

Comment:
This and the line above is meant to work for both the notify banner and the preference centre; couldn't see how to split that preference, seems to be "display on both" or "hide on both" (but see previous line's comment)